jwst/wfs_combine/wfs_combine.py

```diff
@@ -591,54 +643,16 @@ def get_overlap(sci_int_1, sci_int_2, nom_off_x, nom_off_y):
 
     # From the nominal offsets, determine array indices to shift image #2
     #     onto frame #1
-    ix, fx = get_index_range(nom_off_x, sci_int_2.shape[1])
-    iy, fy = get_index_range(nom_off_y, sci_int_2.shape[0])
+    initial_x_1, final_x_1, initial_x_2, final_x_2 = get_index_range(nom_off_x, sci_int_2.shape[1])
+    initial_y_1, final_y_1, initial_y_2, final_y_2 = get_index_range(nom_off_y, sci_int_2.shape[0])
 
-    sub_1 = sci_int_1[:fy - iy, :fx - ix]
-    sub_2 = sci_int_2[iy:fy, ix:fx]
+    sub_1 = sci_int_1[initial_y_1:final_y_1, initial_x_1:final_x_1]
+    sub_2 = sci_int_2[initial_y_2:final_y_2, initial_x_2:final_x_2]
 
     return sub_1, sub_2
 
 
-def optimize_offs(sci_nai_1, sci_nai_2):
```

Comment:
This original scheme was the one requested by the WFS&C team quite a few years ago. Does the new cross-correlation scheme below perform essentially the same way, and if not, has the change been approved by the WFS&C team?

Reply:
Hi @hbushouse - @mwregan2 is now part of the WFS&C team, and has our 👍 for these changes. Thanks for checking! These improvements are in response to issues we've encountered during the various WF Team Practices and LRE sims.